core/src/validator.rs

```diff
@@ -201,6 +206,33 @@ impl Validator {
 
         let blocktree = Arc::new(blocktree);
 
+        if config.enable_ctrl_c_handler {
+            let bank_forks = bank_forks.clone();
+            let counter = Arc::new(AtomicUsize::new(0));
+            set_handler(move || {
+                if counter.fetch_add(1, Ordering::Relaxed) > 0 {
+                    warn!("Validator aborting on second ^C");
```

Comment:
I think we can remove this `warn!` entirely, it doesn't really add much

Reply:
i have deleted warn! here